locales/PT_br.json

```
@@ -0,0 +1,139 @@
+{
+	"nav": {
+		"about": "Sobre",
+		"faq": "FAQ",
+		"credits": "Créditos",
+		"progress": "Progresso"
+	},
+	"hero": {
+		"subtitle": "Servidores de jogos",
+		"title": "Recreated",
+		"text": "Pretendo é uma reposição gratuíta e de código aberto para os servidores de ambos 3DS e Wii U, permitindo conectividade online para todos, mesmo após os servidores originais serem descontinuados",
+		"buttons": {
+			"readMore": "Mostrar mais"
+		}
+	},
+	"aboutUs": {
+		"title": "Sobre nós",
+		"paragraphs": [
+			"Pretendo é um projeto de código aberto que visa recriar a Nintendo Network para o 3DS e Wii U usando design de sala limpa.",
+			"Como os nossos serviços serão gratuitos e de código aberto, eles podem continuar existindo por muito mais tempo após o inevitável encerramento da Nintendo Network."
+		]
+	},
+	"progress": {
+		"title": "Progresso",
+		"paragraphs": [
+			"Atualmente estamos trabalhando no Miiverse, juntamente com nossos servidores de contas e sua integração com os serviços.",
+			"Para o 3DS também estamos trabalhando no Mario Kart 7, com o desejo de trabalhar em outros jogos quando possível."
+		]
+	},
+	"faq": {
+		"title": "Perguntas frequentes",
```

You should change it to this, as to keep it consistent with other locales:
```suggestion
		"title": "Perguntas frequentes (FAQ)",
```